### PR TITLE
Issue 7 error in 11.0 releases

### PR DESCRIPTION
**Pull request by TommyBigNose**
Changed integration test Constants.cs to TestConstants.cs
Updated packages to match latest Relativity Release (contract issues)
Added optional parameters when testing locally vs elsewhere

### CODE REVIEW

File: SourceCode/SmokeTest.Tests/SmokeTest.Tests.csproj [hunk 1]
```diff
@@ -34,29 +34,32 @@
...
       <HintPath>..\packages\BCrypt-Official.0.1.109\lib\BCrypt.Net.dll</HintPath>
     </Reference>
-    <Reference Include="DbContextHelper, Version=3.0.0.2, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\RelativityDev.DbContextHelper.4.0.0.1\lib\net462\DbContextHelper.dll</HintPath>
+    <Reference Include="BCrypt.Net-Core, Version=1.6.0.0, Culture=neutral, PublicKeyToken=4d63bfbfdfd3c6e4, processorArchitecture=MSIL">
+      <HintPath>..\packages\BCrypt.Net-Core.1.6.0\lib\net452\BCrypt.Net-Core.dll</HintPath>
     </Reference>
-    <Reference Include="kCura, Version=10.1.199.4, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Relativity.Other.10.1.199.4\lib\net462\kCura.dll</HintPath>
+    <Reference Include="DbContextHelper, Version=5.0.0.1, Culture=neutral, processorArchitecture=MSIL">
```
Comment (calimetikcura) [inline SourceCode/SmokeTest.Tests/SmokeTest.Tests.csproj]: I thought we removed the DbContextHelper dependency from DevVM. @TommyBigNose Can you please remove the nuget package from the solution and compile and verify it's not being consumed anywhere.

File: SourceCode/SmokeTest/SmokeTestCollection.cs [hunk 2]
```diff
@@ -74,6 +73,7 @@ public class SmokeTestCollection
 			RelativityUrl = relativityUrl;
 			InstanceSettingManager = instanceSettingManager;
 			AuditObjectManagerUiService = auditObjectManagerUiService;
+			_ipAddressForVisualStudio = ipAddressForVisualStudio; // We pass this Ip Address when running tests from Visual Studio
```
Comment (calimetikcura) [inline SourceCode/SmokeTest/SmokeTestCollection.cs]: can we mention in the comment that this ipAddress is for the instance/DevVM where the tests are running to be more specific.

**Review by calimetikcura — CHANGES_REQUESTED** [review]

File: SourceCode/SmokeTest.Tests/SmokeTest.Tests.csproj (hunk 1 above)
Comment (TommyBigNose) [reply SourceCode/SmokeTest.Tests/SmokeTest.Tests.csproj]: Updated, no longer using this, tested with updates on the removal

File: SourceCode/SmokeTest/SmokeTestCollection.cs (hunk 2 above)
Comment (TommyBigNose) [reply SourceCode/SmokeTest/SmokeTestCollection.cs]: Updated with more detail

**Review by calimetikcura — APPROVED** [review]

**Pull request merged**